xapi-xenopsd.opam

```diff
@@ -7,28 +7,28 @@ dev-repo: "https://github.com/xapi-project/xenopsd.git"
 bug-reports: "https://github.com/xapi-project/xenopsd/issues"
 build: [
   ["./configure"]
-  [ "jbuilder" "build" "-p" name "-j" jobs ]
+  [ "dune" "build" "-p" name "-j" jobs ]
 ]
-build-test: [[ "jbuilder" "runtest" "-p" name "-j" jobs ]]
+build-test: [[ "dune" "runtest" "-p" name "-j" jobs ]]
 depends: [
-  "jbuilder" {build}
+  "dune" {build}
+
   "cohttp"
   "core"
   "ezxenstore"
-  "fd-send-recv" {>= "2.0.0"}
-  "ounit" {test}
-  "qmp"
+  "fd-send-recv"
+  "forkexec"
+  "ounit"
+  "ppx_sexp_conv"
   "re"
-  "rpc"
+  "rpclib"
   "sexplib"
+  "stdext"
+  "base-threads"
   "uri"
   "uuidm"
   "uutf"
-  "xapi-forkexecd"
-  "xapi-idl"
-  "xapi-inventory"
-  "xapi-stdext-unix"
-  "xenctrl"
+  "xcp"
```

Comment:
I think we wanted to use `xapi-idl` instead here, but the public name of the library is still xcp for compatibility (otherwise we'd have to update all dune files in many repos to use `xapi-idl` instead of `xcp`.
Not sure whats the best course of action here, but keeping dune and opam synchronized and consistent looks like the way to go, when we rename the package in dune we can do so in opam too, meanwhile maintaining an exception for `xapi-idl` would be more complicated than just using xcp.